packages/salesforcedx-vscode-apex/package.json

```diff
@@ -105,7 +106,6 @@
         "@salesforce/apex-tmlanguage": "1.4.0",
         "@salesforce/core": "7.3.1",
         "@salesforce/source-tracking": "6.0.4",
-        "@salesforce/templates": "^60.1.2",
```

Comment:
I believe this is a typo. templates has never been used by apex and it even does not exist in the project's dependency.